kafka/coordinator/consumer.py

```diff
@@ -731,7 +731,12 @@ def _send_offset_fetch_request(self, partitions):
         for tp in partitions:
             topic_partitions[tp.topic].add(tp.partition)
 
-        if self.config['api_version'] >= (0, 8, 2):
+        if self.config['api_version'] >= (0, 10, 2):
+            request = OffsetFetchRequest[3](
+                self.group_id,
+                list(topic_partitions.items())
+            )
+        elif self.config['api_version'] >= (0, 8, 2):
```

Comment:
@royfranco 
I think that in order to keep this client (kafka-python) coherent, according to [this](https://github.dev/dpkp/kafka-python/blob/b68f61d49556377bf111bebb82f8f2bd360cc6f7/kafka/conn.py#L1182)
you might need to change your code a bit:
According to the link I attached, the broker version 0.10.2 correlates to OffsetFetchRequest_v2 (a.k.a OffsetFetchRequest[2]).
So, it might be better to separate the `>` and the `=`:
```python
if self.config['api_version'] > (0, 10, 2):
    ...OffsetFetchRequest[3](...)
elif self.config['api_version'] == (0, 10, 2):
    ...OffsetFetchRequest[2](...)
elif self.config['api_version'] >= (0, 8, 2):
    ....
```